freshmaker/views.py

```diff
@@ -398,6 +398,19 @@ def _validate_rebuild_request(request):
     if not isinstance(data.get('force', False), bool):
         return json_error(400, 'Bad Request', '"force" must be a boolean.')
 
+    if data_get('bundle_related_image_overrides', False) and not container_images:
+        return json_error(
+            400,
+            'Bad Request',
+            'Manual image overriding allowed only when "container_images" is set explicitly'
+        )
+
+    if not isinstance(data.get('bundle_related_image_overrides', False), dict):
+        return json_error(
+            400, 'Bad Request', '"bundle_related_image_overrides" must be a dictionary'
+        )
+    
+
```

Comment:
```suggestion
    
```

Reply:
Github is not showing any change in your suggestion. Is there anything you wanted to discuss about the checks in `view.,py`?

Reply:
It was just a suggestion of removing the redundant blank line, but it seems github didn't render the suggestion correctly.